src/services/billsService.js

```diff
@@ -2,5 +2,8 @@ import api from 'config/api';
 
 export default {
   getBills: () => api.get('listado_de_facturas'),
-  getLastBill: () => api.get('ultima_factura')
+  getLastBill: () => api.get('ultima_factura'),
+  unsucribeDigitalBill: data => api.delete('factura_digital', data),
+  suscribeDigitalBill: data => api.post('factura_digital', data),
```

Comment:
```suggestion
  suscribeDigitalBill: data => api.post('factura_digital', email),
```
Podriamos darle un poco más de semántica que no sea un generic `data` asi se lee mejor!